rllib/utils/test_utils.py

```diff
@@ -1279,7 +1279,12 @@ def check(self, rtol=None):
         # Compare dummy outputs by exact values given that all nets received the
         # same input and all nets have the same (dummy) weight values.
         for v in self.output_values.values():
-            check(v, self.output_values[main_key], rtol=rtol or 0.002)
+            check(
+                v,
+                self.output_values[main_key],
+                atol=atol or 0.0001,
```

Comment:
So if I call `check(rtol=1)`, this will end up checking with `atol=0.0001`?

Reply:
Good point. I think the handling of the atol and rtol args are "suboptimal" inside `check`. Maybe we should just fix the atol=0.0005 in `ModelChecker.check` and that's it. Keep it simple :)

Reply:
done, simplified